lte/gateway/docker/mme/Dockerfile.ubuntu18.04

```diff
@@ -82,7 +82,7 @@ RUN  git clone --recurse-submodules -b v1.15.0 https://github.com/grpc/grpc && \
      git clone https://github.com/cpp-redis/cpp_redis.git && \
      wget https://ftp.gnu.org/gnu/nettle/nettle-2.5.tar.gz && \
      wget https://www.gnupg.org/ftp/gcrypt/gnutls/v3.1/gnutls-3.1.23.tar.xz && \
-     git clone https://liblfds.org/git/liblfds && \
+     git clone git://liblfds.org/liblfds && \
```

Comment:
@lucaaamaral 

does it work for you ? because for me it does not:

```bash
$ git clone git://liblfds.org/liblfds
Cloning into 'liblfds'...
fatal: unable to connect to liblfds.org:
liblfds.org[0: 185.46.59.113]: errno=Connection timed out
liblfds.org[1: 2a00:1128:200:4:0:56ff:fee8:6acc]: errno=Network is unreachable
```

and w/ verbose:

```
$ git clone --verbose git://liblfds.org/liblfds
Cloning into 'liblfds'...
Looking up liblfds.org ... done.
Connecting to liblfds.org (port 9418) ... fatal: unable to connect to liblfds.org:
liblfds.org[0: 185.46.59.113]: errno=Connection timed out
liblfds.org[1: 2a00:1128:200:4:0:56ff:fee8:6acc]: errno=Network is unreachable
```

Reply:
Yes, it is working for me.

Reply:
I have put up a fix in https://github.com/magma/magma/pull/15536/files

   http_archive(
        name = "liblfds",
        urls = ["https://liblfds.org/downloads/liblfds%20release%207.1.0%20source.zip"],
        strip_prefix = "liblfds",
	build_file = "//bazel/external:liblfds.BUILD",
    )

Reply:
@lucasgonze 

thx. I will try from home. Maybe something is blocked in Eurecom network.

Reply:
I am only able to clone the repository using the git url:

```bash
$ git clone  git://liblfds.org/liblfds 
Cloning into 'liblfds'...
remote: Enumerating objects: 1025, done.
remote: Counting objects: 100% (1025/1025), done.
remote: Compressing objects: 100% (1002/1002), done.
remote: Total 1025 (delta 555), reused 0 (delta 0), pack-reused 0
Receiving objects: 100% (1025/1025), 522.98 KiB | 370.00 KiB/s, done.
Resolving deltas: 100% (555/555), done.
```

While the https url returns 500 error:

```bash
$ git clone https://liblfds.org/git/liblfds
Cloning into 'liblfds'...
fatal: unable to access 'https://liblfds.org/git/liblfds/': The requested URL returned error: 500
```

Reply:
While running tests in different scenarios, another unrelated problem has been detected, included at the pull request: https://github.com/magma/magma/pull/15539 .

Reply:
@lucaaamaral @panyogesh 

OK. I have to combine your modifications to make it work.

@panyogesh 

could we have a dedicated PR for `bazel/cpp_repositories.bzl` modification?